### ISSUE TITLE
ortools/networkx not included in requirements

### ISSUE THREAD

**Issue by jGaboardi** [pysal/spopt#10]
Google's [`ortools`](https://developers.google.com/optimization/) is used in [`BaseClass.py`](https://github.com/pysal/spopt/blob/master/spopt/BaseClass.py), but it is not listed in [`requirements.txt`](https://github.com/pysal/spopt/blob/master/requirements.txt) leading to a failure in [TravisCI](https://travis-ci.com/pysal/spopt/jobs/227577306#L900):

```
======================================================================
ERROR: Failure: ImportError (No module named 'ortools')
```

**Comment by jGaboardi** [comment]
Addressing within #9

**Comment by jGaboardi** [comment]
`networkx` not present in `requirements.txt`

**Comment by jGaboardi** [comment]
closed via #9
